### PR TITLE
Feature deploy vercel

### PR DESCRIPTION
**Pull request by lukasz-bien-rmtcfm-com**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,55 @@
...
+
 ## Deployment
 
+To make deploys work, you need to create free account in https://redislabs.com/try-free/, create Redis instance with `RedisJson` module and get informations - REDIS_ENDPOINT_URI and REDIS_PASSWORD. You must pass them as environmental variables.
+
 ### Google Cloud Run
 
 [![Run on Google
-Cloud](https://deploy.cloud.run/button.svg)](https://deploy.cloud.run/?git_repo=https://github.com/RemoteCraftsmen/redis-shopping-cart.git)
+Cloud](https://deploy.cloud.run/button.svg)](https://deploy.cloud.run/?git_repo=https://github.com/RemoteCraftsmen/redis-shopping-cart.git&revision=feature-deploy-vercel)
```
Comment (rafal-ksiazek-rmtcfm-com) [inline README.md]: revision?

File: client/README.md [hunk 2]
```diff
@@ -3,12 +3,24 @@
...
-# copy file and set proper data inside
-cp .env.example .env
+# Environmental variables
+
+Copy `.env.example` to `.env` file and fill environmental variables
+
+-   VUE_APP_API_URL: Backend API URL (default: http://localhost:3000)
+
+
+# Run docker compose or install redis with RedisJson module manually
```
Comment (rafal-ksiazek-rmtcfm-com) [inline client/README.md]: you don't need to install docker in client

File: server/README.md [hunk 3]
```diff
@@ -9,16 +9,23 @@
 ## Development
 
 ```
-# copy file and set proper data inside
-cp .env.example .env
+# Environmental variables
+
```
Comment (rafal-ksiazek-rmtcfm-com) [inline server/README.md:13]: Go into `server` folder and then:

File: server/README.md [hunk 4]
```diff
@@ -9,16 +9,23 @@
...
+Copy `.env.example` to `.env` file and fill environmental variables
+
+-   REDIS_PORT: Redis port (default: 6379)
+-   REDIS_HOST: Redis host (default: 127.0.0.1)
+-   REDIS_PASSWORD: Redis password (default: demo)
+
+# Run docker compose or install redis with RedisJson module manually
```
Comment (rafal-ksiazek-rmtcfm-com) [inline server/README.md]: or go to https://redislabs page and choose free redis (something like this)

File: README.md [hunk 5]
```diff
@@ -1,16 +1,55 @@
...
 
+## Hot to run it locally?
+
+### Prerequisites
+
+- Node - v12.19.0
+- NPM - v6.14.8
+- Docker - v19.03.13 (optional)
+
+### Installation
```
Comment (rafal-ksiazek-rmtcfm-com) [inline README.md]: Maybe: `Local installation`

File: server/README.md (hunk 3 above)
Comment (rafal-ksiazek-rmtcfm-com) [reply server/README.md:13]: Ahh.. never mind - you are in server folder :D

File: README.md [hunk 6]
```diff
@@ -1,16 +1,55 @@
...
+
+```
+# Environmental variables
+
```
Comment (rafal-ksiazek-rmtcfm-com) [inline README.md:21]: Go to `server` folder and then:

File: README.md (hunk 1 above)
Comment (lukasz-bien-rmtcfm-com) [reply README.md]: done

File: client/README.md (hunk 2 above)
Comment (lukasz-bien-rmtcfm-com) [reply client/README.md]: done

File: server/README.md (hunk 3 above)
Comment (lukasz-bien-rmtcfm-com) [reply server/README.md:13]: done

File: server/README.md (hunk 4 above)
Comment (lukasz-bien-rmtcfm-com) [reply server/README.md]: done

File: README.md (hunk 5 above)
Comment (lukasz-bien-rmtcfm-com) [reply README.md]: done

File: README.md (hunk 6 above)
Comment (lukasz-bien-rmtcfm-com) [reply README.md:21]: done (a bit differently so with client is the same)

File: server/README.md [hunk 7]
```diff
@@ -9,16 +9,25 @@
 ## Development
 
 ```
-# copy file and set proper data inside
-cp .env.example .env
+# Environmental variables
+
+Go into client folder and then:
```
Comment (rafal-ksiazek-rmtcfm-com) [inline server/README.md]: no no no no no

Comment (rafal-ksiazek-rmtcfm-com) [reply server/README.md]: remove this line

**Review by rafal-ksiazek-rmtcfm-com — APPROVED** [review]

File: server/README.md (hunk 7 above)
Comment (lukasz-bien-rmtcfm-com) [reply server/README.md]: done

**Pull request merged**